mfr/extensions/md/render.py

```diff
@@ -23,13 +14,10 @@ class MdRenderer(extension.BaseRenderer):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.metrics.add('markdown_version', markdown.version)
 
     def render(self):
         """Render a markdown file to html."""
-        with open(self.file_path, 'r') as fp:
-            body = markdown.markdown(fp.read(), extensions=[EscapeHtml()])
-            return self.TEMPLATE.render(base=self.assets_url, body=body)
+        return self.TEMPLATE.render(base=self.assets_url, url=self.metadata.download_url)
```

Comment:
MD renderer is now of type **direct-to-wb** instead of **through-renderer**. Only pass the WB download URL to the template.